app/controllers/posts_controller.rb

```diff
@@ -13,7 +13,6 @@ def new
   end
 
```

Comment:
- [ ] Let's mark the _PostsController_ class actions with CanCan load authorization validation by adding `load_and_authorize_resource` at the top of the class. Kindly ensure to improve upon this aspect, similar to:

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```